react-commenting.js

```diff
@@ -63,6 +67,7 @@ class Board extends Component{
       const [a, b, c] = value
       if(gameBoard[a] && gameBoard[a] === gameBoard[b] && gameBoard[a] === gameBoard[c]){
         // 6) Here:
+        // Set state for the current player
         this.setState({
```

Comment:
That is true, but specifically it is setting the state of winner to be the current player.